modules/ROOT/pages/understanding-enterprise-integration-patterns-using-mule.adoc

```diff
@@ -0,0 +1,92 @@
+= Understanding Enterprise Integration Patterns Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Enterprise Integration Patterns are accepted solutions to recurring problems within a given context. The patterns provide a framework for designing, building messaging and integration systems, as well as a common language for teams to use when architecting solutions.
+
+Mule supports most of these patterns shown in the Enterprise Integration Patterns book written by Gregor Hohpe and Bobby Woolf.
+
+By implementing these patterns, Mule reduces the effort required when building integrations. You can use these well-known patters to design your solution and then simply configure and use these same patters in Mule.
+
+
+== Mapping Enterprise Integration Patterns into Mule Objects
+
+There is a list of patterns that can be mapped directly with Mule objects. Review the following list of Enterprise Integration Patterns mapped to Mule objects:
+
+=== Integration Styles
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|File Transfer |xref:connectors::file/file-connector.adoc[File Connector]
+|Shared Database |xref:connectors::db/db-connector-index.adoc[DataBase Connector]
+|Remote Procedure Invocation | Mule APIs are meant to work like this (or even doing requests to external APIs).
```

Comment:
Can you clarify what this means: "work like this"? Like what? 
Also this: "or even doing requests"? Do you mean "sending requests to external APIs"? And how does this relate to "work like this"?